xbmc/pvr/PVRManager.cpp

```diff
@@ -627,15 +627,16 @@ bool CPVRManager::Load(void)
   /* start the add-on update thread */
   if (m_addons)
     m_addons->Start();
+  else
+    return false;
 
-  /* load at least one client */
-  while (IsInitialising() && m_addons && !m_addons->HasConnectedClients())
-    Sleep(50);
```

Comment:
The focus is to fix this properly, not to shove something in at the last moment that only works half.


-------- Oorspronkelijk bericht --------
Van: Sam Stenvall <notifications@github.com> 
Datum:30-04-2015  11:42  (GMT+01:00) 
Aan: xbmc/xbmc <xbmc@noreply.github.com> 
Cc: Lars Op den Kamp <opdenkamp@gmail.com> 
Onderwerp: Re: [xbmc] [pvr] finally make it possible to use multiple PVR clients
  (#7020) 

In xbmc/pvr/PVRManager.cpp:

>  
> -  /* load at least one client */
> -  while (IsInitialising() && m_addons && !m_addons->HasConnectedClients())
> -    Sleep(50);
Since the focus of this PR is to get something in that works, can we agree on a suitable timeout and focus on refactoring this properly some time in the future?

—
Reply to this email directly or view it on GitHub.